core-web/libs/contenttype-fields/src/lib/fields/binary-field/binary-field.component.scss

```diff
@@ -1,24 +1,22 @@
 @use "variables" as *;
 
+:host {
+    display: block;
+    container-type: inline-size;
+    container-name: binaryField;
+}
+
 ::ng-deep {
```

Comment:
this `::ng-deep` not need to be inside the `:host`?

Reply:
I'll move it